generated/library_versions.json

```diff
@@ -0,0 +1,3 @@
+{
+    "org.scala-lang:scala3-library_3": "3.4.1"
```

Comment:
There's only `scala-library_3` to include? if so there's no reason for separate JSON. Just use the `scala_version.txt`.

Reply:
Well, it includes some common library that I'd argue should've been in the standard library.

On second thought, it's fine to do it this way, would make it easier if we do decide to include more modules.